.github/workflows/merge-pr.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   changes:
     name: Determine Changed Files
     runs-on: ubuntu-20.04
-    container: ghcr.io/libertydsnp/frequency/ci-base-image
+    container: ghcr.io/libertydsnp/frequency/ci-base-image:1.0.0
```

Comment:
Would it be worth making this an Actions variable instead? I guess an update to the version always requires a code change anyway, but it would be easier to roll back. Hmm... Simple and explicit is perhaps better.

Reply:
I wouldn't worry about it then. This a good solution to 🚢 I think.